### ISSUE TITLE
Wrong number of followers in mail

### ISSUE THREAD

**Issue by THausherr** [thepracticaldev/dev.to#6955]
**Describe the bug**

Followers count in mail is off by one

**To Reproduce**

- Have a new follower
- get a mail
You have 1 new DEV followers in the past day!
XXX just followed you 🤗
You now have 48 total followers. See all of your recent followers right here.

- click on the link
it says "Followers(49)"

- wait 24 hours to make sure that nobody else followed
still 49 followers, no further mail

**Expected behavior**

The correct follower count in the mail (I checked by entering CTRL-F @ in the browser, I really have 49 followers)

![grafik](https://user-images.githubusercontent.com/6665575/77820860-bb467500-70e5-11ea-87a4-297be0d32ab7.png)

![grafik](https://user-images.githubusercontent.com/6665575/77820869-cf8a7200-70e5-11ea-9f9b-a008bad54cc9.png)

**Desktop (please complete the following information):**

- OS: W10
- Browser: FF
- Version: 74.0

**Comment by citizen428** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
I don't think this is a bug, I think the count simply increased by one between you receiving the email and watching the counter on the dashboard.

The count in the email is computed by:

https://github.com/thepracticaldev/dev.to/blob/40310bbc59934d47ad0e3c3c13404fbf719ece08/app/views/mailers/notify_mailer/new_follower_email.html.erb#L9

The count in the dashboard is computed by:

https://github.com/thepracticaldev/dev.to/blob/40310bbc59934d47ad0e3c3c13404fbf719ece08/app/views/dashboards/_actions.html.erb#L7-L8

as you can see, it uses the same exact variable. 

The email won't be sent for each new follower, to avoid spamming users:

https://github.com/thepracticaldev/dev.to/blob/40310bbc59934d47ad0e3c3c13404fbf719ece08/app/workers/follows/send_email_notification_worker.rb#L10-L12

Basically what that means is "don't send an email if sometime between the last 15 and 35 hours the user has already been sent one". That's why you didn't receive a second email.

Let me know if that makes sense @THausherr and thanks for asking! :)

**Comment by THausherr** [comment]
Thank you for the explanation. I don't even understand why 49 people are following me despite me never having posted anything, only comments. 🤣
